composer.json

```diff
@@ -33,15 +33,14 @@
         "spryker/zed-request": "^3.8"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.0",
-        "propel/propel": "^2.0@alpha",
+        "propel/propel": "2.0.0-alpha12",
```

Comment:
We cannot go to latest propel, but this one is working correctly for production Spryker 2021.08